mixer/adapter/dogstatsd/config/config.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package adapter.dogstatsd.config;
+
+import "gogoproto/gogo.proto";
+
+option go_package="config";
+
+message Params {
```

Comment:
Can we add some documentation here?  This will form the basis of the website docs, so it would be good to have thorough docs here.

Reply:
Ah didn't realize that. I'll expand on the comments.